src/models/MyAccount.ts

```diff
@@ -114,18 +131,25 @@ export default class MyAccount {
   buildHolds(holds: SierraHold[], bibData: SierraBibEntry[]): Hold[] {
     const bibDataMap = MyAccount.buildBibData(bibData)
     return holds.map((hold: SierraHold) => {
+      // Hold without bibIds is a bib level id.
+      const bibId = hold.record.bibIds ? hold.record.bibIds[0] : hold.record.id
```

Comment:
checking for bibIds to indicate if something is an item level hold works, but can you instead use `hold.recordType === 'i'` just to be little more explicit?

Reply:
What is `hold.recordType`? It doesn't seem to be on the holds that I'm looking at

Reply:
it's a string, in our case either `"i"` or `"b"`. I'm seeing it on every hold that comes back from the Sierra API.